SSToastMessage/Sources/Classes/MessageView.swift

```diff
@@ -95,6 +111,12 @@ public struct MessageView<MessageContent>: ViewModifier where MessageContent: Vi
             } else {
                 return -presenterContentRect.midY + sheetContentRect.height/2 + verticalPadding
             }
+        case .leftRightToastView:
+            if position == .bottom {
+                return viewHeight/2 - toastViewHeight/2 - 30
```

Comment:
`viewHeight/2 - toastViewHeight/2` is used twice. Please use a private computed property.